ui-automation-tests/support/steps/register-steps.ts

```diff
@@ -39,6 +39,8 @@ Then("the error message with the text {string} must be displayed for the {} fiel
 
 async function selectOption(page: Page, option: string, value: string) {
     const radio = await page.$x(`//div[@id="${option}-options"]//input[@value="${value}"]`);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
It looks like Line 42 is disabling Line 43 so can't you just remove both?

Reply:
we need this for now David due to linting